core/vm/instructions.go

```diff
@@ -382,6 +387,14 @@ func opSAR(pc *uint64, interpreter *EVMInterpreter, contract *Contract, memory *
 	return nil, nil
 }
 
+func hashSha3(data []byte) []byte {
+	hasherBuf := make([]byte, 64)
```

Comment:
So are these actually nibbles and not bytes? Wouldn't the result be 32-bytes? I'm probably missing something 😄

Reply:
Good eye! Nah, you're right. This should be 32. I've been using Ed25519 keys on my last project which are 64 bytes and inadvertently typed 64 instead of 32. Looking into this lead me to find that Geth already has it's own `Keccak256`/`Sha3` function which we can use instead. Even better! 😄  -> [Use crypto.Keccak256 instead of our own hash function](https://github.com/op-optimism/go-ethereum/pull/1/commits/c52fa1ae5aa8ea18c2ea80cbd0321ad9ed62c3e7)